src/commands/community/create.ts

```diff
@@ -0,0 +1,118 @@
+import { CommunityCreateResource } from '../../shared/community/connect/CommunityCreateResource';
+import { CommunityCreateResponse } from '../../shared/community/defs/CommunityCreateResponse';
+
+Messages.importMessagesDirectory(__dirname);
+const messages = Messages.loadMessages('@salesforce/plugin-community', 'create');
+
+const MESSAGE_KEY = 'message';
+const NAME_KEY = 'name';
+const ACTION_KEY = 'action';
+
+/**
+ * A command to create a community.
+ * This is just an sfdx wrapper around the community create connect endpoint
+ */
+export class CommunityCreateCommand extends SfCommand<CommunityCreateResponse> {
+  public static readonly aliases = ['force:community:create'];
+  public static readonly summary = messages.getMessage('summary');
+  public static readonly description = messages.getMessage('description');
+  public static readonly examples = messages.getMessage('examples').split(os.EOL);
+  public static readonly flags = {
+    name: Flags.string({
+      char: 'n',
+      summary: messages.getMessage('flags.name.description'),
+      description: messages.getMessage('flags.name.longDescription'),
+      required: true,
+    }),
+    'template-name': Flags.string({
+      char: 't',
+      summary: messages.getMessage('flags.templateName.description'),
+      description: messages.getMessage('flags.templateName.longDescription'),
+      required: true,
+      aliases: ['templatename'],
+    }),
+    'url-path-prefix': Flags.string({
+      char: 'p',
+      summary: messages.getMessage('flags.urlPathPrefix.description'),
+      description: messages.getMessage('flags.urlPathPrefix.longDescription'),
+      required: true,
+      aliases: ['urlpathprefix'],
+    }),
+    description: Flags.string({
+      char: 'd',
+      summary: messages.getMessage('flags.description.description'),
+      description: messages.getMessage('flags.description.longDescription'),
+    }),
+    'target-org': requiredOrgFlagWithDeprecations,
+    loglevel,
+    'api-version': Flags.orgApiVersion(),
+  };
+
+  public static readonly validationPatterns: string[] = [
+    // Exact matches
+    'name',
+    'urlPathPrefix',
+    'templateName',
+    'description',
+
+    // templateParams.*, but must be only word characters (e.g. no spaces, special chars)
+    'templateParams(\\.\\w+)+',
+  ];
+  private logger: Logger;
+
+  public async run(): Promise<CommunityCreateResponse> {
+    this.logger = Logger.childFromRoot(this.constructor.name);
+    const { flags, argv } = await this.parse(CommunityCreateCommand);
+    const varargs = this.parseVarargs(argv);
+    const createCommand = new CommunityCreateResource({
+      name: flags.name,
+      urlPathPrefix: flags['url-path-prefix'],
+      templateName: flags['template-name'],
+      description: flags.description,
+      templateParams: varargs['templateParams'] as JsonMap,
+    });
+    return new ConnectExecutor(createCommand, flags['target-org']).callConnectApi().then((results) => {
```

Comment:
this pattern (passing an org to the executor) was benefiting from sfdxCommand already setting the apiVersion on the org.  As is, it's ignoring the version flag

with SfCommand, you'd need to either
1. call getConnection(flags['api-version']) so that the version is set for the Executor to correctly use
2. pass the api-version to the executor
3. pass the executor a connection instead of an org (from a quick glance,that's what it seems to want anyway)